.eslintrc.json

```diff
@@ -0,0 +1,294 @@
+{
+    "env": {
+        "node": true,
+        "es2022": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "overrides": [
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins": [
+        "@typescript-eslint"
+    ],
+    "rules": {
+		"constructor-super":             "warn",
+		"for-direction":                 "warn",
+		"getter-return":                 "warn",
+		"no-async-promise-executor":     "error",
+		"no-await-in-loop":              "error",
+		"no-class-assign":               "error",
+		"no-compare-neg-zero":           "off",
+		"no-cond-assign":               ["error", "always"],
+		"no-const-assign":               "error",
+		"no-constant-binary-expression": "error",
+		"no-constant-condition":         "error",
+		"no-constructor-return":         "error",
+		"no-control-regex":              "warn",
+		"no-debugger":                   "error",
+		"no-dupe-args":                  "error",
+		"no-dupe-class-members":         "error",
+		"no-dupe-else-if":               "error",
+		"no-dupe-keys":                  "error",
+		"no-duplicate-case":             "off",
+		"no-duplicate-imports":          "error",
+		"no-empty-character-class":      "error",
+		"no-empty-pattern":              "error",
+		"no-ex-assign":                  "warn",
+		"no-fallthrough":                "off",
+		"no-func-assign":                "warn",
+		"no-import-assign":              "error",
+		"no-inner-declarations":        ["error", "both"],
+		"no-invalid-regexp":             "error",
+		"no-loss-of-precision":          "error",
+		"no-misleading-character-class": "error",
+		"no-new-native-nonconstructor":  "error",
+		"no-new-symbol":                 "error",
+		"no-obj-calls":                  "error",
+		"no-promise-executor-return":    "warn",
+		"no-prototype-builtins":         "off",
+		
+		"accessor-pairs":             "off",
+		"arrow-body-style":          ["warn", "as-needed"],
+		"block-scoped-var":           "error",
+		"camelcase":                  "error",
+		"capitalized-comments":       "off",
+		"class-methods-use-this":     "off",
+		"complexity":                ["warn", 20],
```

Comment:
```suggestion
		"complexity":                ["off", 20],
```
Beat me

Reply:
we can increase it, but not turn it off
write better code

Reply:
invalid point.

Reply:
A high complexity decreases code readability and maintainability.

Reply:
nagut macht sinn

Reply:
```suggestion
		"complexity":                ["warn", 40],
```

Reply:
nice try.
just reduce code complexity by 11 in POP3Server.ts
we can do 24 if that makes you happy

Reply:
30

Reply:
Not a negotiation.
20 is the default in eslint. 24 is already 20% more

Reply:
fine :rolling_eyes:

Reply:
```suggestion
		"complexity":                ["warn", 24],
```